src/program.js

```diff
@@ -123,11 +125,21 @@ export class Program {
   }
 }
 
+declare var WEBEXT_BUILD_ENV: string;
```

Comment:
It might be worth adding a comment above this explaining how this is generated by webpack and that you can see how it's generated in `webpack.config.js`.